docs/source/release_notes/release-notes.rst

```diff
@@ -44,6 +44,10 @@ develop
          - Cassandra now attempts to truncate when performing a ``deleteRange(RangeRequest.All())`` in an effort to build up less garbage.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1617>`__)
 
+    *    - |fixed|
```

Comment:
I'm leaving as fixed since the heap size was supposed to go in 0.34, but didn't make it.